### PR TITLE
planner_cspace: publish internally used maps as OccupancyGrid

### LINKED ISSUE #395: planner_cspace: publish some debug data as OccupancyGrid
remembered costmap and hysteresis map is better to be published as OccupancyGrid.

### PR DESCRIPTION
**Pull request by at-wat**
fix #395

### CODE REVIEW

**Comment by at-wat** [comment]
## [[#1128](https://travis-ci.org/at-wat/neonavigation/builds/598521373)] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 7 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_map_organizer_rostest.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_pointcloud_to_maps_rostest.xml: 1 tests
build/test_results/map_organizer/rosunit-test_map_organizer.xml: 4 tests
build/test_results/map_organizer/rosunit-test_pointcloud_to_maps.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/obj_to_pointcloud/rostest-test_test_obj_to_pointcloud_rostest.xml: 1 tests
build/test_results/obj_to_pointcloud/rosunit-test_obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 4 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap_performance.xml: 1 tests
build/test_results/planner_cspace/gtest-test_costmap_bbf.xml: 2 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_astar.xml: 5 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_boundary_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 3 tests
build/test_results/planner_cspace/rosunit-test_navigate_boundary.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 1 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 109 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [[#1129](https://travis-ci.org/at-wat/neonavigation/builds/598556098)] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 7 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_map_organizer_rostest.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_pointcloud_to_maps_rostest.xml: 1 tests
build/test_results/map_organizer/rosunit-test_map_organizer.xml: 4 tests
build/test_results/map_organizer/rosunit-test_pointcloud_to_maps.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/obj_to_pointcloud/rostest-test_test_obj_to_pointcloud_rostest.xml: 1 tests
build/test_results/obj_to_pointcloud/rosunit-test_obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 4 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap_performance.xml: 1 tests
build/test_results/planner_cspace/gtest-test_costmap_bbf.xml: 2 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_astar.xml: 5 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_debug_outputs_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_boundary_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_debug_outputs.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 3 tests
build/test_results/planner_cspace/rosunit-test_navigate_boundary.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 1 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 112 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [[#1130](https://travis-ci.org/at-wat/neonavigation/builds/598556515)] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 7 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_map_organizer_rostest.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_pointcloud_to_maps_rostest.xml: 1 tests
build/test_results/map_organizer/rosunit-test_map_organizer.xml: 4 tests
build/test_results/map_organizer/rosunit-test_pointcloud_to_maps.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/obj_to_pointcloud/rostest-test_test_obj_to_pointcloud_rostest.xml: 1 tests
build/test_results/obj_to_pointcloud/rosunit-test_obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 4 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap_performance.xml: 1 tests
build/test_results/planner_cspace/gtest-test_costmap_bbf.xml: 2 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_astar.xml: 5 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_debug_outputs_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_boundary_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_debug_outputs.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 3 tests
build/test_results/planner_cspace/rosunit-test_navigate_boundary.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 1 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 112 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [[#1130](https://travis-ci.org/at-wat/neonavigation/builds/598556515)] PASSED on melodic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 7 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_map_organizer_rostest.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_pointcloud_to_maps_rostest.xml: 1 tests
build/test_results/map_organizer/rosunit-test_map_organizer.xml: 4 tests
build/test_results/map_organizer/rosunit-test_pointcloud_to_maps.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/obj_to_pointcloud/rostest-test_test_obj_to_pointcloud_rostest.xml: 1 tests
build/test_results/obj_to_pointcloud/rosunit-test_obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 4 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap_performance.xml: 1 tests
build/test_results/planner_cspace/gtest-test_costmap_bbf.xml: 2 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_astar.xml: 5 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_debug_outputs_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_boundary_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_debug_outputs.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 3 tests
build/test_results/planner_cspace/rosunit-test_navigate_boundary.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 1 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 112 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
@DaikiMaekawa please take a look.

File: neonavigation_launch/launch/demo.launch [hunk 1]
```diff
@@ -3,6 +3,13 @@
   <arg name="use_path_with_velocity" default="true" />
   <arg name="vel" default="0.5" />
 
+  <rosparam>
+    planner_3d:
+      remember_updates: true
+      hist_ignore_range: 0.0
+      hist_ignore_range_max: 1.5
+  </rosparam>
```
Comment (DaikiMaekawa) [inline neonavigation_launch/launch/demo.launch]: I know this works fine, but shouldn't these parameters be located inside `<node pkg="planner_cspace" type="planner_3d" name="planner_3d">`?

`planner_3d` namespace is not constant.

File: planner_cspace/src/planner_3d.cpp [hunk 2]
```diff
@@ -760,29 +752,63 @@ class Planner3dNode
...
           point.x = x;
           point.y = y;
-          switch (debug_out_)
-          {
-            case DEBUG_HYSTERESIS:
-              if (cost_estim_cache_[p] == FLT_MAX)
-                continue;
+          if (cost_estim_cache_[p] == FLT_MAX)
+            continue;
+          point.z = cost_estim_cache_[p] / 500;
```
Comment (DaikiMaekawa) [inline planner_cspace/src/planner_3d.cpp]: where does `500` comes from?

Comment (at-wat) [reply planner_cspace/src/planner_3d.cpp]: It's a magic parameter of the visualization used in the master branch. In this PR, it just keep compatibility with the previous behaviour.

File: neonavigation_launch/launch/demo.launch (hunk 1 above)
Comment (at-wat) [reply neonavigation_launch/launch/demo.launch]: `navigation.launch` was originally a skeleton code for real usage, and `demo.launch` is for demonstration which should not be in `navigation.launch`.
However, currently `navigation.launch` is much specific for demonstration. It maybe better to directly put it into `neonavigation_launch/config/navigate.yaml`

Comment (at-wat) [reply neonavigation_launch/launch/demo.launch]: I'm going to move these parameters to `config/navigation.yaml` for now and reorganize `neonavigation_launch` in #398.

**Comment by at-wat** [comment]
## [[#1132](https://travis-ci.org/at-wat/neonavigation/builds/598983454)] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 7 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_map_organizer_rostest.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_pointcloud_to_maps_rostest.xml: 1 tests
build/test_results/map_organizer/rosunit-test_map_organizer.xml: 4 tests
build/test_results/map_organizer/rosunit-test_pointcloud_to_maps.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/obj_to_pointcloud/rostest-test_test_obj_to_pointcloud_rostest.xml: 1 tests
build/test_results/obj_to_pointcloud/rosunit-test_obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 4 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap_performance.xml: 1 tests
build/test_results/planner_cspace/gtest-test_costmap_bbf.xml: 2 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_astar.xml: 5 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_debug_outputs_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_boundary_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_debug_outputs.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 3 tests
build/test_results/planner_cspace/rosunit-test_navigate_boundary.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 1 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 112 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [[#1133](https://travis-ci.org/at-wat/neonavigation/builds/598988092)] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 7 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_map_organizer_rostest.xml: 1 tests
build/test_results/map_organizer/rostest-test_test_pointcloud_to_maps_rostest.xml: 1 tests
build/test_results/map_organizer/rosunit-test_map_organizer.xml: 4 tests
build/test_results/map_organizer/rosunit-test_pointcloud_to_maps.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/obj_to_pointcloud/rostest-test_test_obj_to_pointcloud_rostest.xml: 1 tests
build/test_results/obj_to_pointcloud/rosunit-test_obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 4 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap_performance.xml: 1 tests
build/test_results/planner_cspace/gtest-test_costmap_bbf.xml: 2 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_astar.xml: 5 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-navigation_rostest__antialias_start_true.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_debug_outputs_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_boundary_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_debug_outputs.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 3 tests
build/test_results/planner_cspace/rosunit-test_navigate_boundary.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 1 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 112 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Pull request merged**